shared/desktop/remote/sync-avatar-props.desktop.tsx

```diff
@@ -2,14 +2,12 @@
 // It listens for avatar related actions and bookkeeps them to send them back over the wire
 import * as I from 'immutable'
 import * as React from 'react'
-import * as SafeElectron from '../../util/safe-electron.desktop'
 import * as Container from '../../util/container'
 import {memoize} from '../../util/memoize'
 
 type OwnProps = {
   usernames: I.Set<string>
   setUsernames: (arg0: I.Set<string>) => void
-  remoteWindow: SafeElectron.BrowserWindowType | null
```

Comment:
all these remote window hocs used to actually have a direct instance of the browerwindow. instead we just use the `windowComponent` and `windowParam` as a unique identifier and delegate to the node process